src/components/portal/portal.tsx

```diff
@@ -78,6 +79,18 @@ export class Portal {
     @Prop()
     public inheritParentWidth = false;
 
+    /**
+     * Set `showArrow` to `true` if you need to show sourse of `limel-popover` or `limel-tooltip`
+     */
+    @Prop()
+    public showArrow = false;
```

Comment:
There seems to be some kind of problem with the Basic Example:

<img width="203" alt="Screenshot showing the tooltip with a misaligned arrow" src="https://user-images.githubusercontent.com/154725/180805366-625dfe9c-82a3-4866-8ed1-80bec40fb3fb.png">

Reply:
Yes as I said, that tooltip attachment bug must be fixed. I think this is due to that bug.
Otherwise, we cannot have arrows for tooltips yet.

Reply:
Hmm… I don't think those are related… if they are, I think we've got something wrong in the tooltip styling anyway, because the `<limel-tooltip>` element is just another element. It's not the element we want to position the tooltip by, so the _position_ of the tooltip may be wrong, but regardless of what the element we position the tooltip by, the tooltip should never look like it does in the screenshot above. If the attachment bug was the only problem, I would expect the tooltip to perhaps look like this:

![image](https://user-images.githubusercontent.com/154725/180971909-742ec317-a53e-4e5e-8336-493dbdc9eeb8.png)

Reply:
I'm not sure if you are still waiting for my response regarding this one or not.
But a short answer:

what popper does with the "arrow" is that it will place it in relation to the trigger element, and based on all those different `openDirection`s. In case of tooltip, since we have that bug, the trigger element is ignore, and the tooltip is attached to `<limel-tooltip />` itself (not the trigger which is a `2.25rem` by `2.25rem` `<limel-button />`. In the picture above, `<limel-tooltip />` is placed just below the button, has no `height` but gets a `width` of 100%  of its parent container. Therefore the arrow is pointing to the bottom-center of that 100%-wide element (as it's considered as the trigger), while the tooltip has the `openDirection` of `bottom-start` (left), and is attaching itself to the left side of the 100%-wide-trigger.

So without that bug, everything would be fine.

Reply:
Ok, nice! I think that bug should be pretty easy to fix actually, now that I (hopefully) understand what's causing it 😊